### PR TITLE
Adding create token type and privileged namespace

### PR DESCRIPTION
**Pull request by paigerube14**
### Description
Needed to add a way to label the created namespace with privileges and change the way the prometheus token is obtained 

```
% ./deploy.sh


    ____  _ __  __        __
   / __ \(_) /_/ /___  __/ /_  ____  ____  ____  ___  _____
  / / / / / __/ __/ / / / __ \/ __ \/ __ \/ __ \/ _ \/ ___/
 / /_/ / / /_/ /_/ /_/ / /_/ / /_/ / /_/ / /_/ /  __/ /
/_____/_/\__/\__/\__, /_.___/\____/ .___/ .___/\___/_/
                /____/           /_/   /_/


Using k8s command: oc
Using namespace: dittybopper
Using default grafana password: admin

Getting environment vars...
Prometheus URL is: https://prometheus-k8s-openshift-monitoring.apps.**.qe.devcluster.openshift.com
Prometheus bearer token collected.

Creating namespace...
Looks like the namespace dittybopper already exists, deleting it
namespace "dittybopper" deleted
namespace/dittybopper created

Deploying Grafana...
service/dittybopper created
route.route.openshift.io/dittybopper created
deployment.apps/dittybopper created
configmap/sc-ocp-prom created
configmap/sc-grafana-config created

Waiting for dittybopper deployment to be available...
deployment.apps/dittybopper condition met

You can access the Grafana instance at http://dittybopper-dittybopper....
```

### Fixes
https://github.com/cloud-bulldozer/performance-dashboards/issues/59

```
12-07 08:01:30.782  Command "get-token" is deprecated, and will be removed in the future version. Use oc create token instead.
12-07 08:01:30.782  error: could not find a service account token for service account "prometheus-k8s"
12-07 08:01:31.074  Command "new-token" is deprecated, and will be removed in the future version. Use oc create token instead.
```

### CODE REVIEW

**Comment by paigerube14** [comment]
@jtaleric @rsevilla87 can you PTAL when you get a chance, thanks!

File: dittybopper/deploy.sh [hunk 1]
```diff
@@ -120,7 +121,7 @@ fi
 echo ""
 echo -e "\033[32mGetting environment vars...\033[0m"
 export PROMETHEUS_URL="https://$($k8s_cmd get routes -n openshift-monitoring prometheus-k8s -o jsonpath="{.spec.host}")"
-export PROMETHEUS_BEARER=$($k8s_cmd sa get-token -n openshift-monitoring prometheus-k8s || $k8s_cmd sa new-token -n openshift-monitoring prometheus-k8s)
+export PROMETHEUS_BEARER=$($k8s_cmd create token -n openshift-monitoring prometheus-k8s --duration=6h || $k8s_cmd sa get-token -n openshift-monitoring prometheus-k8s || $k8s_cmd sa new-token -n openshift-monitoring prometheus-k8s)
```
Comment (jtaleric) [inline dittybopper/deploy.sh:124]: Curious about the duration here?

Comment (paigerube14) [reply dittybopper/deploy.sh:124]: using the same timing as e2e-benchmark [here](https://github.com/cloud-bulldozer/e2e-benchmarking/blob/dfa9d314b3ea05217c5debd8518f590ec4411f0d/workloads/kube-burner/common.sh#L12). Can increase it to 12h

**Review by jtaleric — APPROVED** [review]

File: dittybopper/deploy.sh (hunk 1 above)
Comment (paigerube14) [reply dittybopper/deploy.sh:124]: Can update with no timeout, but this is the standard we have used in other repos. Thoughts?

Comment (afcollins) [reply dittybopper/deploy.sh:124]: I have clusters that stay up longer than 6 hours. I still vote to remove it. Raised a separate PR.

**Pull request merged**